src/mdoc/checkCallback.ts

```diff
@@ -3,20 +3,60 @@ import { MDLError } from './errors';
 
 const log = debug('mdl');
 
+export const VerificationAssessmentId = {
```

Comment:
Would be helpful this to be exported from the package to make the ID checking bit simpler.